File: LeetCode/twosum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        ans =[]
        for indx, val in enumerate(nums):
            needed = target - val
            for i in nums:    
                if(i==needed and nums.index(i) != indx):
                    ans.append(indx)
                    ans.append(nums.index(i))
                    return ans

File: LeetCode/TwoSum.js
var twoSum = (nums, target) => {
  const ans = [];
  for (const [index, element] of nums.entries()) {
    var needed = target - element;
    if (nums.includes(needed) && nums.indexOf(needed) != index) {
      ans.push(index);
      ans.push(nums.indexOf(needed));
      return ans;
    }
  }
};
